recipe/meta.yaml

```diff
@@ -9,8 +9,7 @@ source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: b9c8559c1ceec118bd8ebafa1c654582fc13d64c66266ca4d15a2df2367be870
 build:
-  number: 0
-  skip: true  # [py<39]
+  number: 1
   script:
     - call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86_amd64   # [win64 and vc==14]
```

Comment:
this looks dubious ....

Reply:
Back then (I do not remember in details), meson python was calling the wrong setup for visual studio so it was a way to force the use of vs14.0.

Now, not sure it is necessary.

Reply:
Lets see